MFVNeutralino/python/EventHistos_cfi.py

```diff
@@ -2,8 +2,6 @@
 
 mfvEventHistos = cms.EDAnalyzer('MFVEventHistos',
                                 mevent_src = cms.InputTag('mfvEvent'),
-                                gen_src = cms.InputTag('prunedGenParticles'),
```

Comment:
Maybe I'm misremembering, but I thought we have some event histo plots that include gen info at times. Does this break the ability to make those?